src/runtime/entries/firebaseV2.ts

```diff
@@ -0,0 +1,11 @@
+import "#internal/nitro/virtual/polyfill";
+// @ts-ignore
+import functions from "firebase-functions/v2";
```

Comment:
Why is a `@ts-ignore` required here?
Also, I believe the best practice nowadays is to use `@ts-expect-error` with a reason.

Reply:
@sisou I'm not sure actually if/why it's needed.... I used the existing firebase entry as boiler for this and it was in there...:
https://github.com/unjs/nitro/blob/main/src/runtime/entries/firebase.ts#L2

Reply:
It's because the 'firebase-functions' package is not installed in nitro. Rather it should be installed by the parent package that is using nitro. I've updated to ts-expect-error with a comment 👍